### PR TITLE
Support presto

### PR DESCRIPTION
**Pull request by eaba**

### CODE REVIEW

File: charts/pulsar/values.yaml [hunk 1]
```diff
@@ -83,13 +83,15 @@ components:
...
+  # pulsar sql
+  presto: true
 
 ## Monitoring Components
 ##
 ## Control what components of the monitoring stack to deploy for the cluster
 monitoring:
   # monitoring - prometheus
-  prometheus: true
+  prometheus: false
```
Comment (sijie) [inline charts/pulsar/values.yaml:94]: Can you leave the setting unchanged?

File: charts/pulsar/values.yaml [hunk 2]
```diff
@@ -609,14 +615,14 @@ broker:
   # use a component name that matches your grafana configuration
   # so the metrics are correctly rendered in grafana dashboard
   component: broker
-  replicaCount: 3
+  replicaCount: 2
```
Comment (sijie) [inline charts/pulsar/values.yaml:618]: keep this setting as default?

Comment (eaba) [reply charts/pulsar/values.yaml]: @sijie , does the new commits satisfy your request?

File: .vs/ProjectSettings.json [hunk 3]
```diff
@@ -0,0 +1,3 @@
+{
```
Comment (sijie) [inline .vs/ProjectSettings.json:1]: Can you exclude `.vs` from the pull request?

File: charts/pulsar/values.yaml [hunk 4]
```diff
@@ -113,6 +115,10 @@ images:
     repository: streamnative/platform
     tag: v1.0.0
     pullPolicy: IfNotPresent
+  presto:
+    repository: eabanonu/presto-pulsar
```
Comment (sijie) [inline charts/pulsar/values.yaml:119]: any reason why you can't use `apachepulsar/pulsar-all`?

Comment (eaba) [reply charts/pulsar/values.yaml:119]: I have tried, there is not documentation on how to deploy pulsar sql this way - the reason for this chart.
#33 is the issue I have encountered. 
What has been done in eabanonu/presto-pulsar is to extract pulsar connector into presto plugin folder.

File: .vs/ProjectSettings.json (hunk 3 above)
Comment (eaba) [reply .vs/ProjectSettings.json:1]: Ok

File: charts/pulsar/values.yaml (hunk 4 above)
Comment (eaba) [reply charts/pulsar/values.yaml:119]: Let me think on this, it might be possible to use pulsar-all given the way it is currently been done

Comment (sijie) [reply charts/pulsar/values.yaml:119]: thank you @eaba!

Comment (sijie) [reply charts/pulsar/values.yaml:119]: Okay. I am fine with a customized image for now. We will look into how to improve this using apachepulsar/pulsar-all image. Thank you @eaba!

Comment (sijie) [reply charts/pulsar/values.yaml:119]: Can you remove `.vs` directory? Then I can merge this.

**Comment by sijie** [comment]
@eaba awesome contributions!

**Pull request merged**